admin/class-h5p-plugin-admin.php

```diff
@@ -646,7 +664,7 @@ private static function asset_handle($path) {
    * @param string $path
    */
   public static function add_script($handle, $path) {
-    wp_enqueue_script(self::asset_handle($handle), plugins_url('h5p/' . $path), array(), H5P_Plugin::VERSION);
+    wp_enqueue_script(self::asset_handle($handle), plugins_url('h5p/' . $path), array(), H5P_Plugin::VERSION, true);
```

Comment:
Any reason we need to change it to always be in the footer? I'm a little worried that this may break something in some themes.